src/components/desktop/MainPane.js

```diff
@@ -1,9 +1,9 @@
 import SearchOverlay from './SearchOverlay'
 
-const SidePane = () => (
+const MainPane = () => (
   <>
```

Comment:
Do we need this fragment here? My understanding is that this component will probably have more children later (otherwise it does not need to exist).

Maybe we could add a `// TODO!  ...` above this?

Reply:
Yes, it'll have more children later! TODO would be nice.